react/components/TopMenu.js

```diff
@@ -186,12 +173,18 @@ class TopMenu extends Component {
 
   renderMobileCategoryMenu = () => (
     <div className="db dn-ns">
-      <ExtensionPoint id="category-menu" mobileMode />
+      <ExtensionPoint
+        id="category-menu"
+        mobileMode
```

Comment:
I think this mobileMode shouldn't be here. The category-menu could get it from render